ui/utilities/alignment/_index.scss

```diff
@@ -9,6 +9,8 @@
  * @modifier
  */
 .nds-align_absolute-center,
-.nds-align--absolute-center {
+.nds-align--absolute-center, 
+.via-nds .nds-align--absolute-center, 
```

Comment:
Hi @siddhartha-majumdar-sf , we should not add via-nds class. it adds automatically

Reply:
Hey @avneshshakya 
So the via_nds is added to the css during rendering?